src/arch/riscv/pagetable_walker.hh

```diff
@@ -163,11 +164,13 @@ namespace RiscvISA
         Port &getPort(const std::string &if_name,
                       PortID idx=InvalidPortID) override;
 
+
```

Comment:
extra newline?

Reply:
fixed. somehow left this line when I deleted an unwanted function declaration (that I had previously added)